src/main/kotlin/org/jetbrains/research/testspark/actions/TestSparkAction.kt

```diff
@@ -115,7 +117,7 @@ class TestSparkAction : AnAction() {
         private val caretOffset: Int = e.dataContext.getData(CommonDataKeys.CARET)?.caretModel?.primaryCaret!!.offset
         private val fileUrl = e.dataContext.getData(CommonDataKeys.VIRTUAL_FILE)!!.presentableUrl
 
-        private val codeTypeButtons: MutableList<JRadioButton> = mutableListOf()
+        private val codeTypeButtons: MutableList<JRadioButton> = mutableListOf() // TODO disallow Line codeType for Kex
```

Comment:
Currently it throws a clear error later on in `KexProcessManager`. Wasn't sure if the simplicity of leaving it like this was better. Otherwise the UI logic will become kex specific kex specific stuff

Reply:
If it is clearly stated in the error, then yes. We can just delete the TODO for now